js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/listGraphs/EditGraphButtons.tsx

```diff
@@ -25,23 +30,26 @@ export const EditGraphButtons = ({
         fullConfig,
         graphName: graph.name
       });
-      window.arangoHelper.arangoNotification(
-        "Graph",
+      notifySuccess(
         `Successfully reset display settings for graph "${graph.name}"`
       );
       onClose();
       return info;
     } catch (e: any) {
-      const errorMessage = e.response.body.errorMessage;
-      window.arangoHelper.arangoError(
-        "Could not reset display settings: ",
-        errorMessage
-      );
+      const errorMessage = e?.response?.body?.errorMessage || "Unknown error";
+      notifyError(`Could not reset graph display settings: ${errorMessage}`);
     }
   };
   return (
     <>
-      <DeleteGraphModal isOpen={isDeleteModalOpen} onClose={onCloseDeleteModal} graph={graph} />
+      <DeleteGraphModal
+        isOpen={isDeleteModalOpen}
+        onClose={onCloseDeleteModal}
+        onSuccess={() => {
+          onClose();
+        }}
```

Comment:
To maintain consistency with the other `isOpen`, `onClose` ... callbacks.

```suggestion
        onSuccess={onClose}
```